src/internal/Subject.ts

```diff
@@ -39,7 +39,9 @@ export class Subject<T> extends Observable<T> implements SubscriptionLike {
     super();
   }
 
-  /**@nocollapse */
+  /**@nocollapse
+   * @deprecated use new Subject() instead
```

Comment:
Hmm. I'm not sure about this deprecation message, as this `create` method does things that cannot be done with the `Subject` constructor alone. Your thoughts, @benlesh ?

Reply:
I was not very sure about this one too because the constructor signature is different. Observable one is more straight forward.

Reply:
I think it's fine. The only thing that `create` did that `new Observable` couldn't was allow `apply` and `call`, which people can get from a simple factory function.